Dockerfile.development

```diff
@@ -8,8 +8,9 @@ FROM java:8-jdk
 
 RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv E56151BF && \
     echo "deb http://repos.mesosphere.com/debian jessie main" | tee /etc/apt/sources.list.d/mesosphere.list && \
+    echo "deb http://repos.mesosphere.com/debian jessie main" | tee -a /etc/apt/sources.list.d/mesosphere.list && \
```

Comment:
Is this line really needed? The previous line adds exactly the same repo to exactly the same file.